plugins/pantheon-photos-publishing-extras/YandexPublishing.vala

```diff
@@ -644,7 +644,7 @@ public class YandexPublisher : Spit.Publishing.Publisher, GLib.Object {
     private void start_web_auth () {
         host.set_service_locked (false);
 
-        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?client_id=%s&response_type=token").printf (client_id));
+        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?CLIENT_ID=%s&response_type=token").printf (CLIENT_ID));
```

Comment:
```suggestion
        web_auth_pane = new WebAuthPane (("http://oauth.yandex.ru/authorize?client_id=%s&response_type=token").printf (CLIENT_ID));
```

I didn't test, but I suppose this `CLIENT_ID` is a part of a query and does not need to (or should not) change?

Reply:
Yes, well spotted, this was autoreplaced - I'll fix.